code/modules/mob/dead/observer/orbit.dm

```diff
@@ -66,30 +69,31 @@
 
 		serialized["ref"] = REF(poi)
 
-		var/mob/M = poi
-		if (istype(M))
-			if (isobserver(M))
-				var/number_of_orbiters = length(M.get_all_orbiters())
+		var/mob/mob_poi = poi
+		if (istype(mob_poi))
+			if(isnewplayer(mob_poi))
+				continue
```

Comment:
These two lines should be moved to `getpois()`. I see no reason why a new player still in the lobby screen should be a point of interest at all.

Reply:
> Quick hotfix to #61508.

> This PR will be turned into a more complete fix later.

Reply:
I'm turning POIs into a subsystem and utilising signal handlers so what is and isn't a POI is controlled from one place. This will hopefully prevent this happening again and provide better oversight potential in the future. This will be SSpois.

I'm making the orbit menu a global datum singleton similar to the crew monitor. It will utilise SSpois to verify if the orbit target is valid or not.

I'm adding unit tests to make sure that nothing which should not be a POI ends up as one again in the future. This should prevent regressions.

There's additional stack_traces and multiple layers of failsafes to make sure things that should not be POIs do not ever accidentally get included as POIs.

Reply:
That's a better solution. Thanks for explaining your plan here.